linux/device.go

```diff
@@ -1139,6 +1141,12 @@ func DeleteDevice(dev *model.Device) (err error) {
 	if err != nil {
 		return err
 	}
+	if(!isFC){
+		err = logoutAndDeleteIscsiTarget(dev)
```

Comment:
Why are we initiating iscsi logout and delete at multiple places in the same workflow.
Please remove this check as we already address it in `cleanupDeviceAndSlaves`

Reply:
As per the review comment, I have reverted the changes of logoutAndDeleteIscsiTarget function call from DeleteDevice function.